coderd/database/queries/insights.sql

```diff
@@ -21,6 +21,86 @@ WHERE
 GROUP BY workspace_agent_stats.user_id, users.username, users.avatar_url
 ORDER BY user_id ASC;
 
+-- name: GetUserActivityInsights :many
```

Comment:
@mafredri Could you please take a look and validate if the query makes sense? If so, I will implement the matching`dbfake` part.

Reply:
Had one comment but otherwise this looks mostly correct to me! Only had the one comment.